src/utilities/focus.ts

```diff
@@ -20,6 +20,20 @@ export function getLastFocusable(
   return getPreviousElement(rootElement, currentElement, true, false, true, includeElementsInFocusZones);
 }
 
+// Attempts to focus the first focusable element. If it successfully focuses it will return true. Otherwise it will return false.
```

Comment:
Use JSDoc, describe the params.

focusFirstChild(rootElement) would seem like a simpler helper.